src/Sylius/Bundle/AdminBundle/Resources/views/Common/_channel.html.twig

```diff
@@ -1 +1,17 @@
-<span class="ui small empty circular label" style="background-color: {{ channel.color }}"></span> {{ channel.name }} ({{ channel.code }})
+{% if channel is defined %}
+    <div class="channel">
+        <span class="ui medium empty circular label channel__item" style="background-color: {{ channel.color|default('#000') }}"></span>
+
+        {% if channel.name is defined %}
+            <span class="channel__item">
+                {{ channel.name }}
+            </span>
+        {% endif %}
+
+        {% if channel.code is defined %}
```

Comment:
What would you say about the unification of this logic, to always present a name and label of the channel? 

Also, then we can just pass the whole object to this template.